src/components/UserActions/UserActions.scss

```diff
@@ -116,6 +117,30 @@ $clickable-blue: #004990;
                         }
                     }
                 }
+
+                .coveo-tooltip-origin1 {
+                    display: none;
+                    position: absolute;
+                    top: -2.75em;
+                    padding: 8px 12px;
+                    background-color: $clickable-blue;
+                    border-radius: 4px;
+                    color: white;
+                    margin-left: 1.5em;
+
+                    &:after {
+                        position: absolute;
+                        top: 100%;
+                        content: '';
+                        left: 1em;
+                        border: 10px solid $clickable-blue;
```

Comment:
We can get rid of $clickable-blue here no? to have just
 `border: 10px solid;`  and the color will keep its default value here before being overwritten in the line after.

Reply:
Or maybe having soemthing like this:
```
  border: 10px solid transparent;
  border-top-color: #004990;
```

Reply:
yup you are right